src/js/views/globals/search/patient-search_views.js

```diff
@@ -0,0 +1,80 @@
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import intl from 'js/i18n';
+
+import Picklist from 'js/components/picklist';
+
+import './patient-search.scss';
+
+const EmptyResultsViews = View.extend({
+  tagName: 'li',
+  className: 'patient-search--no-results',
+  template: hbs`{{ @intl.globals.search.patientSearchViews.picklistEmptyView.noResults }}`,
+});
+
+const TipView = View.extend({
+  tagName: 'li',
+  className: 'patient-search--no-results',
+  template: hbs`{{ @intl.globals.search.patientSearchViews.picklistEmptyView.searchTip }}`,
+});
+
+const PatientSearchPicklist = Picklist.extend({
+  className: 'picklist patient-search__picklist',
+  isSelectlist: true,
+  placeholderText: intl.globals.search.patientSearchViews.patientSearchPicklist.placeholderText,
+  getItemFormat(item) {
+    return `${ item.get('first_name') } ${ item.get('last_name') }`;
+  },
+  itemTemplate: hbs`
+    <a class="patient-search__picklist-item{{#if isSelected}} is-selected{{/if}}">
+      {{matchText text query}}{{~ remove_whitespace ~}}
+      <span class="picklist-item__birthdate">{{formatMoment birth_date "MM/DD/YYYY"}}</span>
+    </a>
+  `,
+  emptyView() {
+    if (this.model.get('query') && this.model.get('query').length > 0) return EmptyResultsViews;
```

Comment:
I'm not sure the `&&` will ever happen here?  or was length supposed to be `> 2`?

Reply:
`query` starts out as `undefined`, but after the user types and then clears it, it will be an empty string. My thought is that if the user has completely cleared their query, we should show the tip view again.

Reply:
an empty string is falsy though so I don't think the `length > 0` is doing anything